### PR TITLE
snort 2.9.13

### PR DESCRIPTION
**Pull request by ghost**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by timesflying** [comment]
I do not understand why all checks have failed. I have the software built from source within brew on my machine and it runs fine for hours now.

**Comment by javian** [comment]
@BrewTestBot test this please

**Comment by timesflying** [comment]
i have tested...the same failures again in postinstall.

**Comment by timesflying** [comment]
I need some input here.
I have no idea on how to proceed from here...the previous version had no problem at all, one would assume a simple version bump should do, yet no. It is always the same issue: the postinstall step apparently fails to be able to copy some file to /usr/local/etc/snort/.
Please, help.
Again, it worked fine on my machine...built from source within brew.

**Comment by alebcay** [comment]
@BrewTestBot test this please!

**Comment by timesflying** [comment]
Still no luck, same post-install problems, Snort 2.9.13 still running strong for more than a month now on my machine as built from within brew using the exact same script as submitted in this pull request without any hickup whatsoever.

**Comment by zbeekman** [comment]
For future reference (so we don't have to keep ramming this through CI):

```
        Warning: The post-install step did not complete successfully
You can try again using `brew postinstall snort`
==> An exception occurred within a child process:
  Errno::EACCES: Permission denied @ rb_sysopen - /usr/local/etc/snort/classification.config
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/fileutils.rb:1394:in `initialize'
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/fileutils.rb:1394:in `open'
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/fileutils.rb:1394:in `block in copy_file'
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/fileutils.rb:1393:in `open'
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/fileutils.rb:1393:in `copy_file'
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/fileutils.rb:487:in `copy_file'
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/fileutils.rb:404:in `block in cp'
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/fileutils.rb:1572:in `block in fu_each_src_dest'
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/fileutils.rb:1588:in `fu_each_src_dest0'
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/fileutils.rb:1570:in `fu_each_src_dest'
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/fileutils.rb:403:in `cp'
/usr/local/Homebrew/Library/Homebrew/extend/pathname.rb:209:in `cp_path_sub'
/usr/local/Homebrew/Library/Homebrew/install_renamed.rb:16:in `cp_path_sub'
/usr/local/Homebrew/Library/Homebrew/formula.rb:1005:in `block (2 levels) in run_post_install'
/usr/local/Homebrew/Library/Homebrew/formula.rb:1003:in `glob'
/usr/local/Homebrew/Library/Homebrew/formula.rb:1003:in `glob'
/usr/local/Homebrew/Library/Homebrew/formula.rb:1003:in `block in run_post_install'
/usr/local/Homebrew/Library/Homebrew/utils.rb:478:in `with_env'
/usr/local/Homebrew/Library/Homebrew/formula.rb:1000:in `run_post_install'
/usr/local/Homebrew/Library/Homebrew/postinstall.rb:18:in `<main>'
==> Caveats
For snort to be functional, you need to update the permissions for /dev/bpf*
so that they can be read by non-root users.  This can be done manually using:
    sudo chmod o+r /dev/bpf*
or you could create a startup item to do this for you.
==> Summary
/usr/local/Cellar/snort/2.9.13: 294 files, 8.1MB
Removing: /Users/brew/Library/Caches/Homebrew/snort--2.9.13.mojave.bottle.tar.gz... (0B)
```

File: Formula/snort.rb [hunk 1]
```diff
@@ -39,6 +39,7 @@ def install
       --enable-reload
       --enable-sourcefire
       --enable-targetbased
+      --disable-control-socket
```
Comment (zbeekman) [inline Formula/snort.rb]: Can you please elaborate on why this config option was added?

Comment (timesflying) [reply Formula/snort.rb]: Because the control socket option is only relevant for Linux systems, not for Macs.

**Comment by zbeekman** [comment]
@timesflying can you please try rebasing your branch onto the latest homebrew-core:master? If all else fails, I'll log into one of the CI workers and investigate the permissions

**Comment by timesflying** [comment]
Unfortunately, no. I have allowed pushing to my branch, just in case.

**Comment by javian** [comment]
@BrewTestBot test this please

**Comment by SMillerDev** [comment]
Still a similar issue:

```
        Warning: The post-install step did not complete successfully
You can try again using `brew postinstall snort`
==> An exception occurred within a child process:
  Errno::EACCES: Permission denied @ rb_sysopen - /usr/local/etc/snort/attribute_table.dtd
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/fileutils.rb:1394:in `initialize'
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/fileutils.rb:1394:in `open'
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/fileutils.rb:1394:in `block in copy_file'
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/fileutils.rb:1393:in `open'
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/fileutils.rb:1393:in `copy_file'
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/fileutils.rb:487:in `copy_file'
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/fileutils.rb:404:in `block in cp'
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/fileutils.rb:1572:in `block in fu_each_src_dest'
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/fileutils.rb:1588:in `fu_each_src_dest0'
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/fileutils.rb:1570:in `fu_each_src_dest'
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/fileutils.rb:403:in `cp'
/usr/local/Homebrew/Library/Homebrew/extend/pathname.rb:209:in `cp_path_sub'
/usr/local/Homebrew/Library/Homebrew/install_renamed.rb:16:in `cp_path_sub'
/usr/local/Homebrew/Library/Homebrew/formula.rb:1005:in `block (2 levels) in run_post_install'
/usr/local/Homebrew/Library/Homebrew/formula.rb:1003:in `glob'
/usr/local/Homebrew/Library/Homebrew/formula.rb:1003:in `glob'
/usr/local/Homebrew/Library/Homebrew/formula.rb:1003:in `block in run_post_install'
/usr/local/Homebrew/Library/Homebrew/utils.rb:478:in `with_env'
/usr/local/Homebrew/Library/Homebrew/formula.rb:1000:in `run_post_install'
/usr/local/Homebrew/Library/Homebrew/postinstall.rb:18:in `<main>'
==> Caveats
For snort to be functional, you need to update the permissions for /dev/bpf*
so that they can be read by non-root users.  This can be done manually using:
    sudo chmod o+r /dev/bpf*
or you could create a startup item to do this for you.
```

**Comment by timesflying** [comment]
I know..yet I have no clue what else I can do. I think the issue lies with CI.

**Comment by scpeters** [comment]
@BrewTestBot test this please

**Comment by zbeekman** [comment]
@BrewTestBot test this please

**Comment by timesflying** [comment]
Given that you removed the badges again you assume everything is fine on your side, or whats the story now????

**Comment by SMillerDev** [comment]
@Brewtestbot test this please

**Comment by javian** [comment]
2.9.14.1 is out now

**Comment by timesflying** [comment]
pullrequest for snort 29141 is out, same story. installs and works fine on my machine.

**Comment by Moisan** [comment]
Will close this one in favor of #43296.

**Pull request closed without merging**